app/src/main/java/io/mochahub/powermeter/graph/GraphFragment.kt

```diff
@@ -51,12 +56,28 @@ class GraphFragment : Fragment() {
         }
 
         viewModel.data.observe(viewLifecycleOwner, Observer {
-            graphAdapter.setData(it ?: emptyList())
+            if (it.size < 5) {
+                showEmptyState()
```

Comment:
Why 5?

Reply:
It was a random number, but from testing-checking, >5 data points make a decent starting point for the graph.